Source/WebCore/dom/Element.cpp

```diff
@@ -390,6 +391,17 @@ void Element::setTabIndexForBindings(int value)
     setIntegralAttribute(tabindexAttr, value);
 }
 
+bool Element::isFormAssociatedCustomElement() const
```

Comment:
We heavily rely on this method, yet it appears quite slow.

@rniwa maybe we should take that last bit of `NodeFlag` to avoid getting the queue from rare data?

Reply:
Yeah, we should probably grab the last bit from NodeFlag for this.

Reply:
Resolved in [3520b5d](https://github.com/WebKit/WebKit/pull/4988/commits/3520b5dbbb73fe3a3ee18339c67f0c38dc3b3c1c) by adding a 1-bit field to `HTMLMaybeFormAssociatedCustomElement`, which currently doesn't increase its `sizeof`, but once we move

```
    bool m_hasDuplicateAttribute { false };
    bool m_displayContentsChanged { false };
```

from `Element`, it might.

Reply:
Oops, things change fast, so I've utilized the last bit in [777c4ed](https://github.com/WebKit/WebKit/pull/4988/commits/777c4edf1384ff8b220f34ae5a04894522fb3f90).